3696/0/about_dice_project.rb

```diff
@@ -1,15 +1,26 @@
+# rubocop:disable Lint/MissingCopEnableDirective
+# rubocop:disable Metrics/LineLength
 require File.expand_path(File.dirname(__FILE__) + '/neo')
 
 # Implement a DiceSet Class here:
 #
+# class DiceSet
+#   code ...
+# end
+# :reek:Attribute and :reek:InstanceVariableAssumption
 class DiceSet
-  attr_reader :values
+  attr_accessor :values
 
   def roll(count)
-    @values = Array.new(count).collect! { |_item| Random.new.rand(1..6) }
+    @values = []
```

Comment:
you can initialize values with Array.new and rand could be called without object.

Reply:
Done